generated/nidcpower/library.py

```diff
@@ -74,374 +74,374 @@ def niDCPower_Abort(self, vi):  # noqa: N802
-                self.niDCPower_Commit_cfunc.argtypes = [ViSession_ctype]  # noqa: F405
-                self.niDCPower_Commit_cfunc.restype = ViStatus_ctype  # noqa: F405
+                self.niDCPower_Commit_cfunc.argtypes = [ViSession]  # noqa: F405
+                self.niDCPower_Commit_cfunc.restype = ViStatus  # noqa: F405
         return self.niDCPower_Commit_cfunc(vi).value
 
     def niDCPower_ConfigureApertureTime(self, vi, channel_name, aperture_time, units):  # noqa: N802
         with self._func_lock:
             if self.niDCPower_ConfigureApertureTime_cfunc is None:
                 self.niDCPower_ConfigureApertureTime_cfunc = self._library.niDCPower_ConfigureApertureTime
-                self.niDCPower_ConfigureApertureTime_cfunc.argtypes = [ViSession_ctype, ViChar_ctype, ViReal64_ctype, ViInt32_ctype]  # noqa: F405
-                self.niDCPower_ConfigureApertureTime_cfunc.restype = ViStatus_ctype  # noqa: F405
+                self.niDCPower_ConfigureApertureTime_cfunc.argtypes = [ViSession, ViChar, ViReal64, ViInt32]  # noqa: F405
+                self.niDCPower_ConfigureApertureTime_cfunc.restype = ViStatus  # noqa: F405
         return self.niDCPower_ConfigureApertureTime_cfunc(vi, channel_name, aperture_time, units).value
 
     def niDCPower_ConfigureDigitalEdgeMeasureTrigger(self, vi, input_terminal, edge):  # noqa: N802
         with self._func_lock:
             if self.niDCPower_ConfigureDigitalEdgeMeasureTrigger_cfunc is None:
                 self.niDCPower_ConfigureDigitalEdgeMeasureTrigger_cfunc = self._library.niDCPower_ConfigureDigitalEdgeMeasureTrigger
-                self.niDCPower_ConfigureDigitalEdgeMeasureTrigger_cfunc.argtypes = [ViSession_ctype, ViChar_ctype, ViInt32_ctype]  # noqa: F405
-                self.niDCPower_ConfigureDigitalEdgeMeasureTrigger_cfunc.restype = ViStatus_ctype  # noqa: F405
+                self.niDCPower_ConfigureDigitalEdgeMeasureTrigger_cfunc.argtypes = [ViSession, ViChar, ViInt32]  # noqa: F405
+                self.niDCPower_ConfigureDigitalEdgeMeasureTrigger_cfunc.restype = ViStatus  # noqa: F405
         return self.niDCPower_ConfigureDigitalEdgeMeasureTrigger_cfunc(vi, input_terminal, edge).value
 
     def niDCPower_ConfigureDigitalEdgePulseTrigger(self, vi, input_terminal, edge):  # noqa: N802
         with self._func_lock:
             if self.niDCPower_ConfigureDigitalEdgePulseTrigger_cfunc is None:
                 self.niDCPower_ConfigureDigitalEdgePulseTrigger_cfunc = self._library.niDCPower_ConfigureDigitalEdgePulseTrigger
-                self.niDCPower_ConfigureDigitalEdgePulseTrigger_cfunc.argtypes = [ViSession_ctype, ViChar_ctype, ViInt32_ctype]  # noqa: F405
-                self.niDCPower_ConfigureDigitalEdgePulseTrigger_cfunc.restype = ViStatus_ctype  # noqa: F405
+                self.niDCPower_ConfigureDigitalEdgePulseTrigger_cfunc.argtypes = [ViSession, ViChar, ViInt32]  # noqa: F405
```

Comment:
is the noqa still needed? given that names are now in the form ViInt32?

Reply:
Answer is no.

```
bin/nidmm/nidmm/library.py:399:62: F405 'ViStatus' may be undefined, or defined from star imports: nidmm.ctypes_types
                self.niDMM_GetCalDateAndTime_cfunc.restype = ViStatus
```

We can fix these easily, I think, but that's a minor thing for a later day.